integrations/gcp/main.py

```diff
@@ -180,8 +185,19 @@ async def feed_events_callback(request: Request) -> Response:
         logger.info(
             f"Got Real-Time event for kind: {asset_type} with name: {asset_name} from project: {asset_project}"
         )
+        resource_configs = typing.cast(
+            GCPPortAppConfig, event.port_app_config
+        ).resources
+        matching_resource_configs = [
+            resource_config
+            for resource_config in resource_configs
+            if (
+                resource_config.kind == asset_type
+                and isinstance(resource_config.selector, GCPResourceSelector)
+            )
+        ]
         asset_resource_data = await feed_event_to_resource(
-            asset_type, asset_name, asset_project, asset_data
+            asset_type, asset_name, asset_project, asset_data, matching_resource_configs
```

Comment:
if there are no matching configs, I'm not sure why we should consider continuing processing, perhaps we can log that the kind does not exist, feel free to let me what you think. This goes on to affect downstream types and precessing too